lib/segment/src/vector_storage/quantized/quantized_multivector_storage.rs

```diff
@@ -201,6 +215,18 @@ where
         }
         sum
     }
+
+    pub fn inner_storage(&self) -> &QuantizedStorage {
```

Comment:
Added getters into quantized multivectors. Function to get original dense quantized vectors and some paramenets